src/Header.js

```diff
@@ -0,0 +1,19 @@
+//just some fontawesome imports
+import { faLeaf } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+function Header () {
+    return(
+        <header>
+            <div className="wrapper">
+                <div className="title">
+                    <h1>Sustainability Tracker</h1>
+                    <h2>Track Your Waste Habits</h2>
+                    <FontAwesomeIcon className="leaf" icon={faLeaf} />
+                </div>
+            </div>
+        </header>
+    )
+}
+
```

Comment:
Might be worth considering adding a clickable scroll down arrow at the bottom of the header as the header takes up 100vh. It would allow the user to move to the Form section